### PR TITLE
Add Quartz 2.3.0 to WSO2 Orbit

### PR DESCRIPTION
**Pull request by pubudu91**
This PR will add Quartz 2.3.0 to WSO2 Orbit.

### CODE REVIEW

File: quartz/2.3.0.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (thusithathilina) [inline quartz/2.3.0.wso2v1/pom.xml]: Need licence header

File: quartz/2.3.0.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    <version>2.3.0.wso2v1</version>
+    <packaging>bundle</packaging>
+
+    <distributionManagement>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 Internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+        <snapshotRepository>
```
Comment (thusithathilina) [inline quartz/2.3.0.wso2v1/pom.xml]: remove snapshot repo

File: quartz/2.3.0.wso2v1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+                <checksumPolicy>ignore</checksumPolicy>
+            </releases>
+        </repository>
+    </repositories>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.quartz-scheduler</groupId>
+            <artifactId>quartz</artifactId>
+            <version>2.3.0</version>
```
Comment (thusithathilina) [inline quartz/2.3.0.wso2v1/pom.xml]: dependencies should be optional

File: quartz/2.3.0.wso2v1/pom.xml (hunk 1 above)
Comment (pubudu91) [inline quartz/2.3.0.wso2v1/pom.xml]: Ack.

File: quartz/2.3.0.wso2v1/pom.xml (hunk 2 above)
Comment (pubudu91) [inline quartz/2.3.0.wso2v1/pom.xml]: Ack.

File: quartz/2.3.0.wso2v1/pom.xml (hunk 3 above)
Comment (pubudu91) [inline quartz/2.3.0.wso2v1/pom.xml]: Ack.

**Pull request merged**